src/routes/login.ts

```diff
@@ -62,9 +63,53 @@ export const createLoginRoute: RouteCreator =
           .catch(() => ({ data: { logout_url: "" } }))
       ).data.logout_url || ""
 
+    const redirectToVerificationFlow = (loginFlow: LoginFlow) => {
+      // we will create a new verification flow and redirect the user to the verification page
+      frontend
+        .createBrowserVerificationFlow({
+          returnTo:
+            (return_to && return_to.toString()) || loginFlow.return_to || "",
+        })
+        .then(({ headers, data: verificationFlow }) => {
+          // we need the csrf cookie from the verification flow
+          res.setHeader("set-cookie", headers["set-cookie"])
+          // encode the verification flow id in the query parameters
+          const verificationParameters = new URLSearchParams({
+            flow: verificationFlow.id,
+            message: JSON.stringify(loginFlow.ui.messages),
+          })
+          // redirect to the verification page with the custom message
+          res.redirect("/verification?" + verificationParameters.toString())
+        })
+        .catch(
+          redirectOnSoftError(
+            res,
+            next,
+            getUrlForFlow(
+              kratosBrowserUrl,
+              "verification",
+              new URLSearchParams({
+                return_to:
+                  (return_to && return_to.toString()) ||
+                  loginFlow.return_to ||
+                  "",
+              }),
+            ),
+          ),
+        )
+    }
```

Comment:
This is not better 😅 

But I guess all of these example repos could use a touch up anyway. So might as well be OOS for this PR.